setup.py

```diff
@@ -109,22 +109,36 @@ def get_git_sha():
         "wtforms-json",
     ],
     extras_require={
-        "athena": ["pyathena>=1.10.8,<1.11"],
-        "bigquery": ["pybigquery>=0.4.10", "pandas_gbq>=0.10.0"],
-        "cors": ["flask-cors>=2.0.0"],
-        "gsheets": ["gsheetsdb>=0.1.9"],
-        "hive": ["pyhive[hive]>=0.6.1", "tableschema", "thrift>=0.11.0, <1.0.0"],
-        "mysql": ["mysqlclient==1.4.2.post1"],
-        "postgres": ["psycopg2-binary==2.8.5"],
-        "presto": ["pyhive[presto]>=0.4.0"],
-        "elasticsearch": ["elasticsearch-dbapi>=0.1.0, <0.2.0"],
-        "druid": ["pydruid>=0.6.1,<0.7"],
-        "hana": ["hdbcli==2.4.162", "sqlalchemy_hana==0.4.0"],
-        "dremio": ["sqlalchemy_dremio>=1.1.0"],
-        "cockroachdb": ["cockroachdb==0.3.3"],
-        "thumbnails": ["Pillow>=7.0.0, <8.0.0"],
-        "excel": ["xlrd>=1.2.0, <1.3"],
-        "prophet": ["fbprophet>=0.6, <0.7"],
+        {
```

Comment:
This should be a `Dict[str, List[str]` however it’s wrapped in an additional set of `{}` which is causing the CI issues.